Source/WebCore/platform/graphics/GlyphBuffer.h

```diff
@@ -164,6 +164,14 @@ class GlyphBuffer {
         setWidth(lastAdvance, WebCore::width(lastAdvance) + width);
     }
 
+    void expandAdvanceToLogicalRight(unsigned index, float width)
+    {
+        ASSERT(index < size());
+        auto& lastAdvance = m_advances[index];
+        setWidth(lastAdvance, WebCore::width(lastAdvance) + width);
+        setX(m_origins[index], x(m_origins[index]) + width);
```

Comment:
You use a temporary for `m_advances[index]`, but not for `m_origins[index]`. Seems like it would be better to be consistent.